kuflow-rest/src/test/java/com/kuflow/rest/util/TaskElementValueAccessorMock.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * The MIT License
+ * Copyright © 2021-present KuFlow S.L.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package com.kuflow.rest.util;
+
+import com.kuflow.rest.model.TaskElementValue;
+import java.util.List;
+
+public class TaskElementValueAccessorMock implements TaskElementValueAccessor {
+
+    private List<TaskElementValue> elementValues;
+
+    @Override
+    public List<TaskElementValue> getElementValues() {
+        return this.elementValues;
+    }
+
+    @Override
+    public void setElementValues(List<TaskElementValue> elementValues) {
+        this.elementValues = elementValues;
```

Comment:
```suggestion
        this.elementValues.clear()
        if (elementValues != null) {
            this.elementValues.addAll(elementValues);
        }
```

Reply:
I put Unmodifiable in the mock to simulate the real world